lib/Zonemaster/LDNS/RR.pm

```diff
@@ -162,6 +164,12 @@ Returns the class of the RR.
 
 Returns a string with the RR in presentation format.
 
+=item compare($other)
+
+Compares two L<Zonemaster::LDNS::RR>.
```

Comment:
Maybe amend this with some details on which things are considered and/or which are ignored in the comparison?